freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/termination/event/recipes/ExecutePreTerminationRecipesFinished.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.freeipa.flow.stack.termination.event.recipes;
+
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.freeipa.flow.stack.termination.event.TerminationEvent;
+
+public class ExecutePreTerminationRecipesFinished extends TerminationEvent {
```

Comment:
could you please add `toString` for this, `ExecutePreTerminationRecipesRequest` and `TerminationEvent`?